src/widgets/about-course/ui/about-course-grid/about-course-grid.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactNode } from 'react';
+import classNames from 'classnames/bind';
+import { Image } from '@/shared/ui/image';
+import { Paragraph } from '@/shared/ui/paragraph';
+import { Subtitle } from '@/shared/ui/subtitle';
+
+import styles from './about-course-grid.module.scss';
+
+export const cx = classNames.bind(styles);
+
+type AboutCourseGridProps = {
+  items: {
+    id: number;
+    title: string;
+    info: string | ReactNode;
+    icon: string;
+  }[];
+};
+
+export const AboutCourseGrid = ({ items }: AboutCourseGridProps) => {
+  return (
+    <div className={cx('about-course-grid')} data-testid="about-course-grid">
+      {items.map(({ id, title, info, icon }) => (
+        <article key={id} className={cx('grid-item')} data-testid="about-course-grid-item">
+          <header className={cx('item-title')}>
+            <Image
+              className={cx('grid-icon')}
+              src={icon}
+              alt={`facts about the course - ${title}`}
+              data-testid="grid-icon"
+            />
```

Comment:
That's just an icons without any usefull information

```suggestion
            <Image
              className={cx('grid-icon')}
              src={icon}
              alt=""
              aria-hidden="true"
              data-testid="grid-icon"
            />
```


<img width="805" alt="Screenshot 2024-09-30 at 18 38 00" src="https://github.com/user-attachments/assets/3974d058-d2f9-421d-951b-4017589fef59">

Reply:
Fixed